segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/rocksdb/Metrics.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
```

Comment:
Please revert changes to the license header.

Reply:
Fixed...though this is a new file so there's nothing to revert actually